snapshot-labs/snapshot
# fix(safe): momoize request of getting code

Fixes #
Memoize request to brovider to get code of address

### Changes 
1. add `useMemoize` from vueuse

### How to test
1. Open list of spaces
2. ensure that `getCode` gets called only once.

### To-Do


### Self-review checklist

### Additional notes or considerations
*_(Include any other relevant information or context that may be helpful for the reviewer)_*
